src/fsharp/tast.fs

```diff
@@ -4511,7 +4519,7 @@ let fullCompPathOfModuleOrNamespace (m:ModuleOrNamespace) =
     CompPath(scoref,cpath@[(m.LogicalName, m.ModuleOrNamespaceType.ModuleOrNamespaceKind)])
 
 // Can cpath2 be accessed given a right to access cpath1. That is, is cpath2 a nested type or namespace of cpath1. Note order of arguments.
-let canAccessCompPathFrom (CompPath(scoref1,cpath1)) (CompPath(scoref2,cpath2)) =
+let inline canAccessCompPathFrom (CompPath(scoref1,cpath1)) (CompPath(scoref2,cpath2)) =
```

Comment:
Please don't add ``inline`` to the compiler unless there's a very specific known and measured perf reason to do it, thanks